Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG GO_VERSION=1.21
+ARG GO_VERSION=latest
```

Comment:
I think we recently changed this from `latest` to a specific version, so that customers wouldn't encounter an error if they used a new release before we'd had time to support it.

Reply:
My thought is that now that we are getting our release out an hour after Go does there's, that we could switch back to latest.